persistent-sqlite/Database/Sqlite.hs

```diff
@@ -468,6 +468,7 @@ bind statement sqlData = do
             PersistList l -> bindText statement parameterIndex $ listToJSON l
             PersistMap m -> bindText statement parameterIndex $ mapToJSON m
             PersistDbSpecific s -> bindText statement parameterIndex $ decodeUtf8With lenientDecode s
+            PersistLiteral l -> bindText statement parameterIndex $ decodeUtf8With lenientDecode l
```

Comment:
I’m confused how this can be the same implementation as PersistDbSpecific

Reply:
see comment below